test/cli/test_cli_mask.py

```diff
@@ -180,3 +181,39 @@ def test_cli_mask_sct_4d():
         # There should be 2 files left since we remove tmp files: the input 4d file and the mask.
         assert len(os.listdir(tmp)) == 2
         assert os.path.isfile(fname_output)
+
+
+def test_cli_mask_mrs():
+    with tempfile.TemporaryDirectory(prefix='st_' + pathlib.Path(__file__).stem) as tmp:
+        runner = CliRunner()
+        # The MRS voxel size and its center coordinates (x, y, z) are extracted from the raw data by first converting
+        # the raw data to a NIfTI file and then reading the header of that file.
+        mrs_raw_data = os.path.join(__dir_testing__, 'ds_mrs', 'sub-1_acq-press-siemens-shim_nuc-H_echo-135_svs.rda')
+        gre = os.path.join(__dir_testing__, 'ds_b0', 'sub-fieldmap', 'fmap', 'sub-1_acq-gre_magnitude1.nii.gz')
+        out = os.path.join(tmp, 'mask_mrs.nii.gz')
+        run_subprocess(['spec2nii', 'rda', mrs_raw_data, '-o', tmp])
+        name_nii, ext = os.path.splitext(mrs_raw_data)
+        nii_path = os.path.join(tmp, 'sub-1_acq-press-siemens-shim_nuc-H_echo-135_svs.nii.gz')
+        nii = nib.load(nii_path)
+        header_raw_data = nii.header
+        position_sag = header_raw_data['qoffset_x']
+        position_cor = header_raw_data['qoffset_y']
+        position_tra = header_raw_data['qoffset_z']
+
+        mrs_center = np.array([position_sag, position_cor, position_tra])
+        mrs_voxel_size = header_raw_data['pixdim'][1:4]
+        mrs_voxel_size_str = ' '.join(map(str, mrs_voxel_size))
+        mrs_center_str = ' '.join(map(str, mrs_center))
```

Comment:
Do not reimplement mask_mrs in the test. Just use what `mrs_center_str` and what `mrs_voxel_size_str` contain. The idea is giving a know input and verify that the output is the same across versions of the code. If you include all or some of the implementation of the API, we might not catch these differences.